geode-core/src/main/java/org/apache/geode/cache/client/internal/ExecutablePool.java

```diff
@@ -120,7 +120,11 @@ Object executeOnQueuesAndReturnPrimaryResult(Op op)
 
   /**
    * Release the connection held by the calling thread if we're using thread local connections
+   *
+   * @deprecated Since Geode 1.10.0. Thread local connections are ignored. Will be removed in future
+   *             major release.
    */
+  @Deprecated
```

Comment:
Since this is an internal class can we just remove the method?

Reply:
There was something internal going on when I first got to this method. I’ll double check and see if it can go now.

Reply:
Looks like I just had it confused with the same method on Pool. Thanks!